docs/IoT.js-API-I2C.md

```diff
@@ -0,0 +1,60 @@
+IoT.js provides I2C module.
+
+## Module: i2c
+
+### Constructor
+
+#### new i2c([address[, options]])
+* `address: Number (Default value is 0x23)`
+* `options: Object`
+
+Option is an object specifying following information:
+* `device: String (Default value is /dev/i2c-1)`
+
+Create I2C wire.
+
+### Methods
+
+#### i2c.scan(callback)
```

Comment:
Is this a function provided by the module (e.g., `i2c.scan(...)`) or a function to be called on an object created with the constructor (e.g., `(new i2c()).scan(...)`)? How to differentiate the two types of methods in the doc?